modules/nextflow/src/main/groovy/nextflow/cli/CmdSecret.groovy

```diff
@@ -99,7 +95,7 @@ class CmdSecret extends CmdBase implements UsageAware {
             }
         }
 
-        println result.join('\n').toString()
+        out.println result.join('\n').toString()
```

Comment:
Why `out.` ?

Reply:
I've used the same "trick" in CmdInfo to read the output

Reply:
Ohh, there's a [better trick](https://github.com/nextflow-io/nextflow/blob/27636f9d27c56d286e1e6173e66d97c971c5d6d3/modules/nextflow/src/test/groovy/nextflow/cli/CmdLogTest.groovy#L41-L45) !

Reply:
changed